### ISSUE TITLE
"Eqw" SGE job status not recognized by nextflow as task failure

### ISSUE THREAD

**Issue by Nick-Eagles** [nextflow-io/nextflow#1480]
### Expected behavior and actual behavior

Expected behavior: pipeline execution should halt with an error code/message indicating that the job scheduler (SGE) has failed to submit the process due to a problem in the .command.run script.

Actual behavior: pipeline execution hangs (indefinitely?), apparently waiting on outputs from processes which have actually failed with SGE queue status "Eqw". Pipeline execution must then be forcibly stopped and re-executed from the start (I don't know of a way to force nextflow to stop without losing ability to resume).

### Steps to reproduce the problem

This may be difficult to reproduce. The source of the error in my pipeline run appeared to be a temporary filesystem issue, where the compute node assigned by the SGE `qsub` command to execute .command.run temporarily failed to access the working directory. As a result, SGE issued the `Eqw` job status, usually indicative of bad command-line arguments (in this case, the `#$ -wd` argument in .command.run did not appear to exist to the compute node). The underlying issue should be able to be reproduced by any improper SGE command-line arguments in the .command.run script. This could likely be produced by specifying bad "clusterOptions" in the config for some process (e.g. `clusterOptions = 'anything invalid'`). 

### Program output 

After I observed the "Eqw" status, I checked what SGE said happened (I entered `qstat -j 447661` for the problematic SGE job ID) to get the following snippet:

```
error reason          1:      01/31/2020 13:49:49 [42086:22939]: error: can't chdir to /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67: No such file or directory
```

I then looked in the nextflow log via `cat .nextflow.log | grep "447661"` which appeared to indicate no issues when nextflow queried SGE for the job statuses:
```
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
~> TaskHandler[jobId: 447661; id: 1371; name: SamtoBam (Prefix: Sample_4045); status: SUBMITTED; exit: -; error: -; workDir: /dcl01/lieber/ajaffe/lab/RNAsp_work/runs/04/b6a51203f019b7eaf908b4e84eae67 started: -; exited: -; ]
```

### Environment 

* Nextflow version: 18.10.1
* Java version: OpenJDK 1.8.0-b13
* Operating system: CentOS 7

### Additional context

The full nextflow log:
[.nextflow.log](https://github.com/nextflow-io/nextflow/files/4141662/default.nextflow.log)

**Comment by DaGaMs** [comment]
Related to #1461

**Comment by pditommaso** [comment]
Solved by #1461.
